scenarios/ecs_privesc_evade_protection/terraform/cloudtrail.tf

```diff
@@ -0,0 +1,6 @@
+# Using CloudTrail for GuardDuty
+resource "aws_cloudtrail" "cloudtrail" {
```

Comment:
Add a depends on for the bucket policy `aws_s3_bucket_policy.trail_bucket_policy`